src/functionHandlers/notarisePdt/validateInputs/validateDocument.ts

```diff
@@ -17,6 +19,17 @@ import {
   DocumentInvalidError,
 } from "../../../common/error";
 import { config } from "../../../config";
+import clinicProviderSchema from "../../../static/clinic-provider-schema.json";
+
+function loadSchema(uri: string) {
+  return axios.get(uri).then((res) => res.data);
+}
+const ajv = new Ajv({
+  allErrors: true,
+  strictTypes: false,
```

Comment:
fyi, using this strictTypes as false because the fhir bundle lite schema doesn't have `type`: `object` in some attributes https://schemata.openattestation.com/sg/gov/moh/fhir/4.0.1/lite-schema.json.

Reply:
> lite schema doesn't have `type`: `object`

Mm I'm not sure what you mean this. Any snippet of the error message you were facing? Just afraid if we disable `strictTypes`, it might also cause other checks to be too lenient.

Reply:
> lite schema doesn't have `type`: `object`

Mm I'm not sure what you mean this. Any snippet of the error message you were facing? Just afraid if we disable `strictTypes`, it might also cause other checks to be too lenient.

Reply:
without this strictTypes option. it's throw the warning in log like this 
```
console.warn
      strict mode: missing type "object" for keyword "properties" at "https://schemata.openattestation.com/sg/gov/moh/fhir/4.0.1/lite-schema.json#" (strictTypes)
```

Reply:
Ok I'll see what I can do to fix `lite-schema.json`.

In the meantime, if it's just a warning, let's re-enable `strictTypes`?

Reply:
Hey @zawmyolatt, I think Ajv has some strict mode options you can set to: `true` | `false` | `"log"`

Can see if it helps to suppress the `missing type "object"` warnings?

Source: https://ajv.js.org/options.html#strict-mode-options